clippy_lints/src/import_rename.rs

```diff
@@ -0,0 +1,97 @@
+use clippy_utils::{diagnostics::span_lint_and_sugg, source::snippet_opt};
+
+use rustc_data_structures::fx::FxHashSet;
+use rustc_errors::Applicability;
+use rustc_hir::{def::Res, Item, ItemKind, UseKind};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_session::{declare_tool_lint, impl_lint_pass};
+use rustc_span::Symbol;
+
+declare_clippy_lint! {
+    /// **What it does:** Checks that any import matching `import_renames` is renamed.
+    ///
+    /// **Why is this bad?** This is purely about consistency, it is not bad.
```

Comment:
IMHO: This is bad for some items because their defined names are too unspecific outside of their defining scope, that is `Value` makes sense to be a JSON value inside `serde_json`, but when you see `Value` in a REST app's source code it could be many other things.

Reply:
Yeah, good point! I'll have this be the text.